petastorm/pytorch.py

```diff
@@ -283,7 +287,11 @@ def __init__(self, reader, batch_size=1, transform_fn=None,
         :param batch_size: the number of items to return per batch; factored into the len() of this reader
         :param transform_fn: an optional callable to convert batches from the reader to PyTorch tensors
         :param shuffling_queue_capacity: Queue capacity is passed to the underlying :class:`tf.RandomShuffleQueue`
-          instance. If set to 0, no suffling will be done.
+          instance. If set to 0, no shuffling will be done.
+        :param inmemory_cache_all: This is an boolean that indicates whether the data should be
```

Comment:
Maybe we should mention that the user should expect batch size smaller then requested on the epoch boundary.